docs/developer_tools/canvas_lint.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: default
+title: Canvas Lint
+parent: Developer Tools
+nav_order: 2
+---
+
+Easol has developed and maintains a gem called easol-canvas, which is used for validating Easol themes (See [RubyGems](https://rubygems.org/gems/easol-canvas) or [GitHub](https://github.com/easolhq/canvas)).
+
+The `canvas lint` command will run the following validation checks:
+
+1. RequiredFilesCheck
+    1. Checks that the required files exist (e.g. `templates/product/index.html`).
+2. ValidHtmlCheck
+    1. Validates all `.html` files contain valid HTML.
+3. ValidLiquidCheck
+    1. Validates all `.html` files contain valid Liquid.
+4. COMING SOON: Validate the schemas in blocks, headers and footers.
```

Comment:
I may be wrong but I _think_ the coming soons are now here:

- RequiredFiles
- ValidHtml
- ValidLiquid
- ValidJson
- ValidSass
- ValidBlockSchemas
- ValidMenuSchema
- ValidFooterSchema
- ValidCustomTypes